src/Nethermind/Nethermind.Blockchain/FullPruning/FullPruner.cs

```diff
@@ -158,6 +170,28 @@ private void SetCurrentPruning(IPruningContext pruningContext)
 
         private bool CanStartNewPruning() => _fullPruningDb.CanStartPruning;
 
+        private const double ChainSizeThresholdFactor = 1.3;
```

Comment:
Would make this a `long` and make it in %, set it to 130, then when using it first multiply by it, then divide by 100. You keep everything integer and avoid floats.